generator/generator.toml

```diff
@@ -0,0 +1,6 @@
+[general]
+library_name = "libhwloc"
+output_file_path = "../src/libhwloc.jl"
+module_name = "LibHwloc"
```

Comment:
I often call this just "API"

Reply:
Should I rename this? I am happy with either -- and just to nit-pick, shouldn't this then be called  "ABI" ? :stuck_out_tongue: